src/Sylius/Component/Core/OrderProcessing/OrderAdjustmentsClearer.php

```diff
@@ -26,6 +26,18 @@
 
     public function __construct(array $adjustmentsToRemove = [])
     {
+        if (0 === func_num_args()) {
+            @trigger_error('Adjustment types should be passed explicitly', E_USER_DEPRECATED);
```

Comment:
We also might want to mention that not passing them is deprecated since 1.2 and will be prohibited by 2.0